apps/backend/app/api/routes/transformations.py

```
@@ -56,6 +57,16 @@ async def preview_transformation(
 
         first_step = request.transformation_steps[0]
 
+        # Validate transformation type against whitelist
+        try:
+            transformation_type = EngineTransformationType(first_step.transformation_type)
```

Ruff (0.14.8)

Local variable `transformation_type` is assigned to but never used

Remove assignment to unused variable `transformation_type`

(F841)